AI_ML/NLP/COUNT_VECTORIZER/COUNT_VECTORIZER.py

```diff
@@ -0,0 +1,26 @@
+import numpy
+
+from sklearn.feature_extraction.text import CountVectorizer
+
+from flojoy import flojoy, DataContainer
+
+
+@flojoy
+def COUNT_VECTORIZER(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+    """The COUNT_VECTORIZER node converts a collection (matrix) of text documents to a matrix of token counts.
+
+    Returns
+    -------
+    ordered_pair DataContainer
+        x -> the feature names
+        y -> the word counts themselves
+    """
+    vectorizer = CountVectorizer()
+
+    X = vectorizer.fit_transform(dc_inputs[0].m)
```

Comment:
This function is expecting a 'm' key in input DC which means it only supports 'dataframe' and 'matrix' type. Can we throw an error with meaningful message if any other types are passed to this node?